cmds/core/strace/strace.go

```diff
@@ -25,15 +25,17 @@ import (
 	"github.com/u-root/u-root/pkg/strace"
 )
 
-var (
-	cmdUsage = "Usage: strace <command> [args...]"
+const (
+	defaultOut = "<stdout>"
+	cmdUsage   = "Usage: strace [-o <outputfile>] <command> [args...]"
 )
 
 func usage() {
 	log.Fatalf(cmdUsage)
 }
 
 func main() {
+	o := flag.String("o", defaultOut, "write output to file")
```

Comment:
I'd just make this `"o", "", "Write output to file (if empty, stdout)"`

Reply:
done